corehq/apps/case_search/utils.py

```diff
@@ -113,4 +113,8 @@ def _add_case_property_queries(self):
             for removal_regex in remove_char_regexs:
                 to_remove = re.escape(removal_regex.regex)
                 value = re.sub(to_remove, '', value)
-            self.search_es = self.search_es.case_property_query(key, value, fuzzy=(key in fuzzies))
+
+            if '/' in key:
+                self.search_es = self.search_es.related_case_property_query(self.domain, key, value)
```

Comment:
I think there might be a better way of achieving this. You would get "for free" the recursive multi-parent lookup if 
you instead did:

```py
if `/` in key:
   self.search_es = self.search_es.xpath_query(self.domain, f'{key} = "{value}"'})
```
This is essentially what is happening in the `case_ids_lookup` function you made anyway. 

You could add this as a method on `CaseSearchES` if you wanted, e.g.:
```py
def related_case_property_query(domain, key, value):
    if '/' not in key:
        raise SomeError
    return self.xpath_query(self.domain, f'{key} = "{value}")
```
although this might be overkill too. 

What do you think? It would also simplify this PR a bunch.

Reply:
Ha, my first implementation of this PR was just [this commit](https://github.com/dimagi/commcare-hq/commit/daecb77374fcc1370f66b1e12764c50e781d4121). I'll take another look.

Reply:
It's all coming back to me now. I moved away from just calling `xpath_query`, because the next step of this work is to support fuzzy search for related case lookups. It seemed easier to extract a single-level related case query, and then add fuzziness to that, than to incorporate fuzziness into `xpath_query`.

Incidentally, if you have thoughts on fuzzy related cases searching, my rough notes are  [here](https://docs.google.com/document/d/1wgBlb-9pvsO4TT_Kfw1kGaYFjr171sRJE_puYmR9EUk/edit).